setupVIP.sh

```diff
@@ -0,0 +1,37 @@
+#!/bin/bash -eu
+
```

Comment:
Any reference to `generic` needs to be made dynamic as the user can override it.

Reply:
Another good point - my initial structure of this was as a script separate from the main `multipass-rke2.sh` to let me iterate on changes and keep it separated. This created a "need" for me to export some variables in `multipass-rke2.sh` so they could be used in the added script. Would you think this better if incorporated into a single script? That would remove the need for exports and still support conditional use if the user desires HA/VIP

Reply:
I'm fine with it being a separate script, just make sure that variables from the main script are used.